api/src/main/java/com/github/retrooper/packetevents/protocol/player/User.java

```diff
@@ -361,4 +369,52 @@ public NBTCompound getWorldNBT(String worldName) {
         String dimensionName = dimension.getDimensionName();
         return dimensionName.isEmpty() ? this.getWorldName(dimension.getId()) : dimensionName;
     }
+
+    /**
+     * Stores the given {@param value} with the given {@param identifier}.
+     * This essentially makes this User object associated with the stored value.
+     * Enables a quick look-up of the associated value with the {@link #getStored(StorageValueId)}
+     * method.
+     *
+     * @param identifier The plugin's constant identifier created from {@link StorageValueId#identifierFor StorageValueId.identifierFor}
+     * @param value The value to store.
+     */
+
+    public void storeRuntime(StorageValueId identifier, Object value) {
+        this.fastUserStorage.store(identifier, value);
+    }
+
+    /**
+     * Stores the given {@param value} with the given {@param identifier}.
+     * This essentially makes this User object associated with the stored value.
+     * Enables a quick look-up of the associated value with the {@link #getStored(TypedStorageValueId)}
+     * method.
+     *
+     * @param identifier The plugin's constant identifier created from {@link TypedStorageValueId#identifierFor TypedStorageValueId.identifierFor}
+     *
+     */
+
+    public void storeRuntime(TypedStorageValueId<?> identifier, Object value) {
```

Comment:
see above, also any reason for not using generics here? would ensure more type-safety at compile-time

Reply:
Oh I guess with the Typed one a generic can be used